apps/omg/lib/omg/typed_data_sign/config.ex

```diff
@@ -0,0 +1,70 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.TypedDataSign.Config do
+  @dialyzer {:nowarn_function, domain_separator: 5}
```

Comment:
why are they referenced here? either fix them, or put them in the dialyzer ignore file.

Reply:
I was unable to quickly suppress dialyzer via `dialyzer-ignores`. Because I've created separate :issue: for this, I'm going to move forward

Reply:
Now you can :)